custom/icds_reports/management/commands/quarter_school_data.py

```diff
@@ -0,0 +1,89 @@
+
+from datetime import date
+from dateutil import parser
+import csv
+from django.core.management.base import BaseCommand
+
+from django.db import connections
+
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+from custom.icds_reports.models.aggregate import AwcLocation
+from dimagi.utils.chunked import chunked
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+
+db_alias = get_icds_ucr_citus_db_alias()
+
+
+def _run_custom_sql_script(command):
+    with connections[db_alias].cursor() as cursor:
+        cursor.execute(command)
+        return cursor.fetchall()
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        state_locations = AwcLocation.objects.filter(aggregation_level=1, state_is_test=0).values('state_id',
+                                                                                                  'state_name')
+        final_state_data = {
+            location['state_id']: {
```

Comment:
should that be `location.state_id`?

Reply:
hey, no, I think it would be this only, as `query_set.values(...)` returns a list of dicts